base/applications/mscutils/eventvwr/lang/pt-PT.rc

```diff
@@ -0,0 +1,216 @@
+BEGIN
+    "?", IDM_ABOUT, ASCII, ALT
+    "/", IDM_ABOUT, ASCII, ALT
+    VK_F2, IDM_RENAME_EVENTLOG, VIRTKEY
+    VK_F5, IDM_REFRESH, VIRTKEY
+END
+
+IDD_EVENTDETAILS_DLG DIALOGEX 0, 0, 266, 240
+STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CENTER | WS_THICKFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
+EXSTYLE WS_EX_CONTEXTHELP
+CAPTION "Detalhes do Evento"
+FONT 8, "MS Shell Dlg"
+BEGIN
+    LTEXT "EvtDetailsCtrl", IDC_STATIC, 0, 0, 266, 215
+    PUSHBUTTON "A&juda", IDHELP, 8, 220, 50, 14
+    DEFPUSHBUTTON "&Fechar", IDOK, 208, 220, 50, 14
+END
+
+IDD_EVENTDETAILS_CTRL DIALOGEX 0, 0, 266, 215
+STYLE DS_SHELLFONT | DS_CONTROL | WS_CHILD | WS_CLIPCHILDREN
+FONT 8, "MS Shell Dlg"
+BEGIN
+    LTEXT "Data:", IDC_STATIC, 8, 5, 31, 8
+    EDITTEXT IDC_EVENTDATESTATIC, 50, 5, 47, 8, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP | ES_AUTOHSCROLL
+    LTEXT "Fonte:", IDC_STATIC, 103, 5, 36, 8
+    EDITTEXT IDC_EVENTSOURCESTATIC, 140, 5, 82, 8, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP | ES_AUTOHSCROLL
+    LTEXT "Hora:", IDC_STATIC, 8, 15, 31, 8
+    EDITTEXT IDC_EVENTTIMESTATIC, 50, 15, 47, 8, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP | ES_AUTOHSCROLL
+    LTEXT "Categoria:", IDC_STATIC, 103, 15, 36, 8
+    EDITTEXT IDC_EVENTCATEGORYSTATIC, 140, 15, 82, 8, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP | ES_AUTOHSCROLL
+    LTEXT "Tipo:", IDC_STATIC, 8, 25, 31, 8
+    EDITTEXT IDC_EVENTTYPESTATIC, 50, 25, 47, 8, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP | ES_AUTOHSCROLL
+    LTEXT "EventID:", IDC_STATIC, 103, 25, 36, 8
+    EDITTEXT IDC_EVENTIDSTATIC, 140, 25, 82, 8, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP | ES_AUTOHSCROLL
+    LTEXT "&Utilizador:", IDC_STATIC, 8, 35, 36, 8
+    EDITTEXT IDC_EVENTUSERSTATIC, 50, 35, 152, 8, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP | ES_AUTOHSCROLL
+    LTEXT "&Computador:", IDC_STATIC, 8, 45, 40, 8
+    EDITTEXT IDC_EVENTCOMPUTERSTATIC, 50, 45, 152, 8, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP | ES_AUTOHSCROLL
+
+    PUSHBUTTON "", IDC_PREVIOUS, 230, 5, 28, 14, BS_ICON
+    PUSHBUTTON "", IDC_NEXT, 230, 21, 28, 14, BS_ICON
+    PUSHBUTTON "", IDC_COPY, 230, 37, 28, 14, BS_ICON
+
+    LTEXT "&Descrição:", IDC_STATIC, 8, 65, 39, 8
+    CONTROL "", IDC_EVENTTEXTEDIT, RICHEDIT_CLASS, ES_MULTILINE | ES_AUTOVSCROLL | ES_READONLY | WS_VSCROLL | WS_BORDER, 8, 76, 250, 60
+
+    LTEXT "Dad&os:", IDC_DETAILS_STATIC, 8, 140, 20, 8
+    CONTROL "&Bytes", IDC_BYTESRADIO, "Button", BS_AUTORADIOBUTTON, 39, 140, 34, 8
+    CONTROL "&Word", IDC_WORDRADIO, "Button", BS_AUTORADIOBUTTON, 77, 140, 34, 8
+    EDITTEXT IDC_EVENTDATAEDIT, 8, 150, 250, 60, ES_MULTILINE | ES_AUTOHSCROLL | ES_AUTOVSCROLL | ES_READONLY | WS_VSCROLL | WS_BORDER
+END
+
+IDD_LOGPROPERTIES_GENERAL DIALOGEX 0, 0, 252, 234
+STYLE DS_SHELLFONT | WS_POPUP | WS_CAPTION
+CAPTION "Geral"
+FONT 8, "MS Shell Dlg"
+BEGIN
+    LTEXT "&Display name:", IDC_STATIC, 7, 9, 60, 8
```

Comment:
Are these strings accidentally left untranslated or do you intend to translate them in this PR?

Reply:
Some of the strings I don't know how to translate in the best way. That's why I chose not to translate the entire dialog.
Thanks

Reply:
OK i found a good solution. Im working on it.